src/praisonai/praisonai/templates/tool_override.py

```
@@ -329,24 +346,32 @@ def create_tool_registry_with_overrides(
+    # / ``tools_sources`` continue to work and are the supported way to load
+    # custom tool modules.
+    if _autoload_tools_enabled():
+        # 4.5. Current working directory tools.py (if exists)
+        cwd_tools_py = Path.cwd() / "tools.py"
+        if cwd_tools_py.exists():
             try:
-                tools = loader.load_from_file(str(tools_py))
+                tools = loader.load_from_file(str(cwd_tools_py))
                 registry.update(tools)
             except Exception:
                 pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)